option/option.go

```diff
@@ -50,3 +51,32 @@ func (o *Option) NewCmd(args ...string) *exec.Cmd {
 	cmd.Env = append(os.Environ(), o.env...)
 	return cmd
 }
+
+// UpdateEnv updates the environment variable for the key name of the input.
+func (o *Option) UpdateEnv(env string) {
```

Comment:
Would it be easier if the function signature took 2 arguments instead? Like:

```
func (o *Option) UpdateEnv(envKey string, envVal string) {
```

That way you don't have to split the string on `=` in `containsEnv`

Reply:
Thanks for review!!! Will update.

Reply:
fixed.